bin/_config.sh

```diff
@@ -58,28 +79,14 @@ function _docker_run() {
         i=$(( i+1 ))
     done
 
-    # Check that the current user is already logged in OpenShift
-    if ${OC_LOGIN} && ! oc whoami &> /dev/null; then
-        echo "Error: you need to login to an OpenShift server first."
-        exit 1
-    fi
-
     [[ "$USE_TTY" == "false" ]] && TTY_OPTION="" || TTY_OPTION="-t"
 
-    # Get the OpenShift host from OC
-    OPENSHIFT_HOST=$(oc version | grep Server | awk '{print $2}')
-
-    # The OpenShift host is of the form https://domain:8443 so we can
-    # extract the domain from it
-    # shellcheck disable=SC2034,SC2001
-    OPENSHIFT_DOMAIN=$(echo "${OPENSHIFT_HOST}" | sed 's#https://\(.*\):8443#\1#')
-
     docker run --rm -i ${TTY_OPTION} \
         -u "$(id -u)" \
         --env-file "$env_file" \
-        --env OC_LOGIN \
-        --env K8S_AUTH_API_KEY="$(oc whoami -t)" \
-        --env K8S_AUTH_HOST="${OPENSHIFT_HOST}" \
+        --env K8S_AUTH_API_KEY \
+        --env K8S_AUTH_HOST \
+        --env OPENSHIFT_HOST \
```

Comment:
redundant with K8S_AUTH_HOST?

Reply:
Indeed, but I find it more explicit, and the other one is required. Should I remove it?

Reply:
I've removed it.